config/playground/helm/dev/helm-opensearch-dashboards.yaml

```diff
@@ -0,0 +1,216 @@
+# Copyright OpenSearch Contributors
```

Comment:
Are you mistakenly adding this file?

Reply:
Yes, that was by mistake. I got it removed now.